scripts/apps/desks/directives/StageItemListDirective.js

```diff
@@ -206,12 +206,18 @@ export function StageItemListDirective(search, api, superdesk, desks, cards, $ti
                 }
             };
             scope.fetchPrevious = function() {
-                if (!scope.fetching && scope.page > 2) {
+                if (!scope.fetching && scope.page > 2 && criteria.source.from > 0) {
                     scope.fetching = true;
                     scope.page -= 1;
-                    criteria.source.from = scope.page > 3 ? (scope.page - 3) * criteria.source.size : 0;
                     scope.loading = true;
 
+                    if (scope.page > 3) {
+                        criteria.source.from = (scope.page - 2) * criteria.source.size;
+                    } else {
+                        criteria.source.size *= 2;
```

Comment:
just tested locally, after some scrolling down and up, from and size are like:
```
prev 80 40
prev 0 80
next 240 80
next 320 80
next 400 80
next 480 80
prev 320 80
prev 240 80
prev 160 80
prev 0 160
next 480 160
next 640 160
next 800 160
```
plus some errors about duplicate items